app/js/stores/QuestionStore.js

```diff
@@ -114,6 +122,10 @@ QuestionStore.dispatchToken = AppDispatcher.register((action) => {
       loadTopQuestions(action.data.questions);
       QuestionStore.emitChange();
       break;
+    case ZhishiConstants.RECEIVE_USER_QUESTIONS:
+      loadUserQuestions(action.data.questions);
+      QuestionStore.emitChange();
+      break;
```

Comment:
using an if inside a case statement doesn't look right. This should be taken care of from the function that is calling the case methods.